test/helpers.ts

```diff
@@ -7,9 +7,13 @@ export const gitLfsVersion = '2.13.2'
 
 const temp = require('temp').track()
 
-export async function initialize(repositoryName: string): Promise<string> {
+export async function initialize(repositoryName: string, defaultBranch?: string): Promise<string> {
   const testRepoPath = temp.mkdirSync(`desktop-git-test-${repositoryName}`)
-  await GitProcess.exec(['init'], testRepoPath)
+  if (defaultBranch) {
+    await GitProcess.exec(['init', '-b', defaultBranch], testRepoPath)
+  } else {
+    await GitProcess.exec(['init'], testRepoPath)
+  }
```

Comment:
I think I prefer this approach:
```suggestion
  const branchArgs = defaultBranch !== undefined ? ['-b', defaultBranch] : []
  await GitProcess.exec(['init', ...branchArgs], testRepoPath)
```

And, in any case, if that `if (defaultBranch)` is intentional (meaning, you're also checking it's not an empty string), I'd rather have that checked explicitly.

Does that sound ok to you?